.travis.yml

```diff
@@ -1,25 +1,23 @@
+## Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
-
 os:
   - linux
   - osx
-
 julia:
-  - 0.6
+  - 0.7
```

Comment:
Please add 1.0 as well

Reply:
I'll change the allow failures to nightly as well?

Reply:
Hopefully there will be a release before julia `v2`. I think we can save the planet a bit and skip nightly for now.